harmonica/forward/prism.py

```diff
@@ -212,6 +234,9 @@ def jit_prism_gravity(coordinates, prisms, density, kernel, out):
     """
     # Iterate over computation points and prisms
     for l in prange(coordinates[0].size):
+        # Update progress bar if called
+        if progress_proxy is not None:
+            progress_proxy.update(1)
```

Comment:
I think it might be better to move these lines to the bottom of the for loop: if for some reason the very last iteration fails, the progressbar will still show 100%. But if we move these lines at the end,reaching the 100% is done only after the last iteration finishes. Do you agree?

Reply:
Good call, I'll move them down.